api-spec/protobuf/ark/v1/admin.proto

```diff
@@ -55,6 +55,23 @@ service AdminService {
       body: "*"
     };
   }
+  rpc GetIntentFees(GetIntentFeesRequest) returns (GetIntentFeesResponse){
+    option (meshapi.gateway.http) = {
+      get: "/v1/admin/intentFees"
+    };
+  }
+  rpc UpdateIntentFees(UpdateIntentFeesRequest) returns (UpdateIntentFeesResponse){
+    option (meshapi.gateway.http) = {
+      post: "/v1/admin/intentFees"
+      body: "*"
+    };
+  }
+  rpc ClearIntentFees(ClearIntentFeesRequest) returns (ClearIntentFeesResponse){
+    option (meshapi.gateway.http) = {
+      post: "/v1/admin/intentFees/clear"
+      body: "*"
+    };
+  }
```

Comment:
Do we really need a clear RPC ? can be an `UpdateIntentFees` with empty strings? Just asking, maybe we need it cc @altafan @Kukks